default.nix

```diff
@@ -209,7 +209,7 @@ let
       client = let
         generated-purescript = pkgs.runCommand "plutus-playground-purescript" {} ''
           mkdir $out
-          ${playground-exe}/bin/plutus-playground-server psgenerator $out
+          ${server-invoker}/bin/plutus-playground psgenerator $out
```

Comment:
Not that it really makes much difference but why the change?

Reply:
The `psgenerator` call now uses the interpreter. It has to, to run the simulations. So it needs the wrapped version with the right `--set GHC_LIB_DIR` etc. flags set.

(I am very glad you figured those flags out long, long ago. :-D )